shared/pubsub.js

```diff
@@ -412,11 +412,6 @@ const defaultMessageHandlers = {
     }, client.options.pingTimeout)
   },
 
-  [NOTIFICATION_TYPE.PUB] ({ channelID, data }) {
-    console.log(`[pubsub] Received data from channel ${channelID}:`, data)
-    // No need to reply.
-  },
-
```

Comment:
I think it's actually a good idea to have this default handler here so that people know it's a built-in one that can be overwritten.